python/dgl/distributed/dist_graph.py

```diff
@@ -339,6 +344,12 @@ def __init__(self, server_id, ip_config, num_servers,
             # Create the graph formats specified the users.
             self.client_g = self.client_g.formats(graph_format)
             self.client_g.create_formats_()
+            if sort_etypes == 'csr':
+                self.client_g = sort_csr_by_tag(
+                    self.client_g, tag=self.client_g.edata[ETYPE], tag_type='edge')
```

Comment:
@jermainewang @zheng-da Is sorting like this correct ? Is passing tag with `g.edata[ETYPE]` correct? Do I need to sort the inner edges only?